hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/utils/TracingContext.java

```diff
@@ -165,6 +165,10 @@ public int getRetryCount() {
     return retryCount;
   }
 
+  public FSOperationType getOpType() {
```

Comment:
General comment: Please add javadocs for public interfaces in this PR.

Reply:
Added javadocs where ever applicable